src/harness/fake_sas.py

```diff
@@ -409,16 +409,36 @@ def do_GET(self):
     self.wfile.write(json.dumps(response))
 
 
-def RunFakeServer(version, is_ecc):
+def RunFakeServer(version, is_ecc, ca_cert, verify_crl):
   FakeSasHandler.SetVersion(version)
   if is_ecc:
     assert ssl.HAS_ECDH
+  if ca_cert is not None:
+    assert os.path.exists(os.path.join('certs',ca_cert)), "%s is not exist in certs path" % ca_cert
   server = HTTPServer(('localhost', PORT), FakeSasHandler)
-  server.socket = ssl.wrap_socket(
+
+  if verify_crl:
+    # If verify CRL flag is set then load the ca chain with CRLs and verify that
+    # the client certificate is not revoked.
+    ssl_context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+    ssl_context.options &= ssl.CERT_REQUIRED
```

Comment:
This means "clear all flags, but leave `CERT_REQUIRED` unchanged."  Did you mean `|=`?

Reply:
Yes, you are right. Thanks for this. Fixed.